### ISSUE TITLE
"Recive" button is not appearing in footer of AR list after filtering by Request Id.

### ISSUE THREAD

**Issue by pioneerpathan** [senaite/senaite.core#822]
## Steps to reproduce

1) After login with labclerk or labmanager role, go to Analysis Request Listing screen.
2) Now select minimum one due sample from list
3) In the footer "Receive" button gets visible.
4) Now, enter "Request Id" of any due sample in to search box located at the header of same listing.
5) It will filter out the records and shows only matching with filter criteria.
6) Now select at least one due sample from filtered list.
7) You would not be able to see "Receive" button in footer of the listing. It should be visible to make "Receive" state transition.

Same is the case for "Cancel" button.

## Expected behavior

"Recieve" and "Cancel" button should visible even after filtering of data for due samples.

## Screenshot (optional)
![ar listing sample due to receive issue](https://user-images.githubusercontent.com/9590337/40115672-dc489b00-592e-11e8-8e67-cacd56e87d80.png)

![ar listing sample due to receive issue filtered record](https://user-images.githubusercontent.com/9590337/40115686-ea782ed4-592e-11e8-94ea-24eae0656362.png)

**Comment by juangallostra** [comment]
@pioneerpathan do you still have this issue? I have tried to replicate it but in my instance it is working as expected. 

This is the list of Analysis Requests without filtering:
![seleccio_026](https://user-images.githubusercontent.com/9968427/41228168-b36591fe-6d77-11e8-8c54-ebcbbf045ab3.png)
And this is the filtered one:
![seleccio_027](https://user-images.githubusercontent.com/9968427/41228170-b609d316-6d77-11e8-8df3-beda3e576583.png)

**Comment by pioneerpathan** [comment]
Hi @juangallostra Thanks for the comments.

I have recheck the issue. It still persists. 

I am using the very first version of the Senaite.Core. I haven't updated it thoroughly after that. Let me know how to check the version of senaite.core I am using.

**Comment by juangallostra** [comment]
@pioneerpathan upgrading the version of `senaite.core` will likely solve the issue. You can check the version you are running by:

1- Go to the **Site Setup** page:
![seleccio_029](https://user-images.githubusercontent.com/9968427/41229473-676bd006-6d7c-11e8-8aa4-eadbb614cbcc.png)

2- Click on the **Zope Management Interface**:
![seleccio_030](https://user-images.githubusercontent.com/9968427/41229474-679adf5e-6d7c-11e8-9a3e-60004d4ca72c.png)

3- Look for **portal_setup** and click on it:
![seleccio_031](https://user-images.githubusercontent.com/9968427/41229475-67c9c01c-6d7c-11e8-94ea-4ede977e9b4a.png)

4- Move to the **Upgrades** tab: 
![seleccio_032](https://user-images.githubusercontent.com/9968427/41229607-ebeb2656-6d7c-11e8-8eaf-88a553e51908.png)

5- In the dropdown profiles list select **bika.lims:default** and click on Choose Profile. 
![seleccio_033](https://user-images.githubusercontent.com/9968427/41229476-67f87ef2-6d7c-11e8-9700-02867c1c08db.png)

**Comment by pioneerpathan** [comment]
Hi @juangallostra Thanks for details for checking the current version. Mine is **1.2.1**, which very first version of senaite.core.

I would surely upgraded to latest releases but I am scared that my local code changes does not get whipped off.

I will create a new instance with latest release 1.2.6 and check the issue that I am facing.

Additionally, It would be really helpful, If you can show me specific changes required to address the issue.

**Comment by juangallostra** [comment]
@pioneerpathan have you been able to check if the problem persists in an instance running the latest version of `senaite.core`? 

And also, do you get any specific traceback in the terminal when filtering the list of Analysis Requests?

**Comment by pioneerpathan** [comment]
@juangallostra I haven't tried with newer instance. I will let you know once I do it.

When filtering the list, I got following traceback in terminal :

``
2018-06-12 15:25:35 WARNING Bika ClientAnalysisRequestsView: sort_on is 'id', not a valid column/index
2018-06-12 15:25:35 INFO Bika Error (Or, DateIndex='created')
2018-06-12 15:25:35 INFO Bika Error (Or, DateIndex='getSamplingDate')
``

**Comment by ramonski** [comment]
Hi @pioneerpathan,
please update to the latest version of `senaite.core`, the issues you described are already fixed in there. Also try to stick to one issue per ticket as it will get confusing for us otherwise.
Thanks

**Comment by pioneerpathan** [comment]
Hi @juangallostra and @ramonski I have checked with latest version of `senaite.core` , the issue is NOT there in latest version and it is working as expected. Issue has been resolved. Thanks.

@ramonski I tried to stick with the issue that I reported in this ticket, anyway apologies for any confusing points. And now onward I will stick with the latest version of `senaite.core`.

Thanks for all your help and support.:+1
